packages/core/core/src/PackagerRunner.js

```diff
@@ -23,6 +23,7 @@ import {
   md5FromString,
   blobToStream,
   TapStream,
+  clearCache,
```

Comment:
could the name make it clearer that this cache refers to the config cache?

Reply:
an alternative might be to add a `clear` method to the loadConfig function itself, e.g.:

```js
await loadConfig('/foo/bar');
loadConfig.clear();
```

Reply:
i like the idea of adding a `clear` method to `loadConfig`, but `loadConfig` currently takes in at least 3 arguments and might end up looking a little clunky when calling clear on it (`loadConfig(fs, filepath, filenames).clear()`) unless we added default values for the arguments?

Reply:
But this would be a method on the function and not on the return value?
```js
function loadConfig(...) {
	...
}
loadConfig.clear = function() {
  ...
}
```